package.json

```diff
@@ -69,7 +69,7 @@
     "isChromatic.d.ts"
   ],
   "scripts": {
-    "build": "clean-package && tsup && clean-package restore",
+    "build": "clean-package ; tsup ; clean-package restore",
```

Comment:
👏

Reply:
When we do a build in production, don't we want this to fail if `clean-package` breaks?

Reply:
Ah yes, I'll revert the first one back to `&&` so tsup won't run if clean-package fails, but if tsup fails we'll still restore.